<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iAutoRigWidget</class>
 <widget class="QWidget" name="iAutoRigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iAutoRig</string>
  </property>
  <widget class="QLineEdit" name="BodyPartName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="HumanoidButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Make Humanoid</string>
   </property>
  </widget>
  <widget class="QLabel" name="BodyPartNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Body Part Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SHAR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateArmButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Arm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SubnetPath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PathTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Drag and drop rig subnet to get path</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateLegButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Leg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateFingerButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Simple FK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateSpineButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Spine</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateHeadAndNeckButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Head and Neck</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateHipButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Hip</string>
   </property>
  </widget>
  <widget class="QLabel" name="UtilitiesLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Utilities</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="MirrorLimbButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mirror Bone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MirrorFingersButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mirror Root</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreatePistonButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Piston</string>
   </property>
  </widget>
  <widget class="QLabel" name="QuickConstraintsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Constraints</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateHandsFingersButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Hand's Fingers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateQuadLegButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Quad Leg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InitializeButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialize</string>
   </property>
  </widget>
  <widget class="QLabel" name="PathTitle_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Single Hierarchy Auto Rigger</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
